src/Cosmos/JsonSerialization.cpp

```diff
@@ -179,6 +179,9 @@ static json messagesJSON(const Proto::SigningInput& input) {
             return json::array();
         } else if ((msg.has_wasm_terra_execute_contract_transfer_message())) {
             j.push_back(messageWasmTerraTransfer(msg.wasm_terra_execute_contract_transfer_message()));
+        } else if ((msg.has_wasm_terra_execute_contract_generic())) {
```

Comment:
Maybe combine the two 'not supported' branches into on, with if (... || ...).

Reply:
makes sense. Will combine